internal/cmd/ocean/spark_create_cluster.go

```diff
@@ -50,6 +46,24 @@ type (
 const (
 	defaultK8sVersion   = "1.21"
```

Comment:
@thorsteinnth 
Is it the most updated version we currently support?

Reply:
I will do a separate PR to update the k8s version and eksctl version